Update README.md

evanscastonguay opened the pull request:
When I try to compile your code did did not work.
I had to commit using the folder above. I am new in go so maybe I just did not have it right; I am just sharing in case other people have similar issue than me.

Commit:
Update README.md

When I try to compile your code did did not work.
I had to commit using the folder above. I am new in go so maybe I just did not have it right; I am just sharing in case other people have similar issue than me.

nikhita commented on README.md:
```diff
@@ -4,6 +4,11 @@ Example repository for the blog post [Kubernetes Deep Dive: Code Generation for
 
+Checkout this repo under this folder and set the GOPATH
+git clone https://github.com/openshift-evangelists/crd-code-generation.git ~/src/github.com/openshift-evangelists/crd-code-generation/
+export GOPATH=~
```
Having a proper GOPATH is necessary while dealing with Go projects and I don't think it's necessary to include it here.

Maybe a "Installing" section would be a better home for this. And just `go get github.com/openshift-evangelists/crd-code-generation` should be enough to install it.

sttts replied:
Isn't there a default GOPATH now with go 1.10?

evanscastonguay replied:
I had it working by git checkout the source under a specific structure (~/src/github.com/openshift-evangelists/crd-code-generation/) and have the GOPATH point to the root of structure. Again this is not a problem for me anymore. But I think that if someone starts using your project that is new in GO make the barrier of entry higher than what is needed. Additional installation instruction would just make it easier for someone new to start using it.

sttts replied:
Can you update the PR with `~/go` as GOPATH? I think that's the most common. Also please use `go get github.com/openshift-evangelists/crd-code-generation` instead of git-clone.

nikhita replied:
> Isn't there a default GOPATH now with go 1.10?

It was there from go 1.8 :) https://golang.org/doc/go1.8#gopath

> Can you update the PR with ~/go as GOPATH? 

What if someone already has the GOPATH set and it is not `~/go`?

sttts replied:
> What if someone already has the GOPATH set and it is not ~/go?

Should be formulated properly to make that clear.

Commit:
update readme file to include installation steps

This commit adds the necessary documentation to explain how to get the package using the go get command.

Commit:
Update README.md

evanscastonguay commented:
ok I update the commit to use get go instead of git clone.
Let me know if this is good enough.

sttts commented on README.md:
```diff
@@ -2,6 +2,11 @@
 
+## Installation
+
+export GOPATH=~/go
```
This should go between \```

sttts commented on README.md:
````diff
@@ -19,7 +24,7 @@ kubectl apply -f artifacts/my-database.yaml
 
 ```
-cd cmd/example
+cd cd ~/go/src/github.com/openshift-evangelists/crd-code-generation/cmd/example/
````
two `cd`. Better do: `cd $GOPATH/src/...`

Commit:
remove duplicate cd

sttts commented:
/lgtm
